### DISCUSSION TITLE
[QUESTION]  Dynamoose Model.batchGet don't support ConsistentRead

### DISCUSSION THREAD

**Discussion by epiphaneSpecter** [dynamoose/dynamoose#1182] [Q&A]
I would like to make batchGet with ConsistentRead support, but in the documentation dynamoose does not support it, dynamoose only offers two properties in the setting object [return and attributes]

I noticed on the aws documentation that dynamodb natively supports it.

So is this an oversight?
And also something that I did not understand very well about the aws documentation, can you do a get with more than 100 items in several steps thanks to the property UnprocessedKeys ?

**Comment by fishcharlie** [comment]
@epiphaneSpecter 

> I would like to make batchGet with ConsistentRead support, but in the documentation dynamoose does not support it, dynamoose only offers two properties in the setting object [return and attributes]

Submit a feature request for this 😃. Please only include this one item in the report.

> And also something that I did not understand very well about the aws documentation, can you do a get with more than 100 items in several steps thanks to the property UnprocessedKeys ?

I actually forget how this behaves off the top of my head. I'd encourage you to try it. I feel like it might give you an error. But not 100% sure.

Might be worth submitting a separate feature request after you've tested it with your findings and suggestions about how Dynamoose can help here. Maybe Dynamoose can make the multiple requests for you or something. That'd be a cool feature.

**Reply by epiphaneSpecter** [reply]
tks @fishcharlie
